apps/webapp/app/services/db/pgListen.server.ts

```diff
@@ -0,0 +1,48 @@
+import { logger } from "~/services/logger.server";
+import { Logger } from "@trigger.dev/core";
+import type { PoolClient } from "pg";
```

Comment:
I've only added types for now. Will leave adding the package for when we actually need it. For now, we only listen to events within `zodWorker` where we can grab an existing `PoolClient`.